installcommands.sh

```diff
@@ -49,31 +50,34 @@ sudo apt-get install -y libtool
 sudo apt-get install -y autoconf
 pip3 install prometheus_client
 pip3 install flask_prometheus_metrics
-pip3 install flas_prometheus_metrics[flask]
+pip3 install flask_prometheus_metrics[flask]
 sudo apt-get install -y arptables
 cd src
 unzip IP2Location-C-Library-master.zip
```

Comment:
Add sudo rm -r IP2Location-C-Library-master before just in case the file is already unzipped. Causes conflicts when this folder is not fresh from the zip